pyvista/__init__.py

```diff
@@ -45,6 +46,15 @@
 except KeyError:
     pass
 
+# If available, a local vtk-data instance will be used for examples
+VTK_DATA_PATH: Optional[str] = None
+if 'PYVISTA_VTK_DATA' in os.environ:
+    VTK_DATA_PATH = os.environ['PYVISTA_VTK_DATA']
+    if not os.path.isdir(VTK_DATA_PATH):
+        warnings.warn(f"VTK_DATA_PATH: {VTK_DATA_PATH} is an invalid path")
+    if not os.path.isdir(os.path.join(VTK_DATA_PATH, 'Data')):
+        warnings.warn(f"VTK_DATA_PATH: {os.path.join(VTK_DATA_PATH, 'Data')} does not exist")
```

Comment:
Agreed with this.  You start in a bad configuration but you should still be allowed to execute code that doesn't require examples.  Only when you actually use it should you raise errors.